src/test/java/it/pagopa/pn/bff/service/TosPrivacyServiceTest.java

```diff
@@ -0,0 +1,170 @@
+package it.pagopa.pn.bff.service;
+
+import it.pagopa.pn.bff.exceptions.PnBffException;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.Consent;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.ConsentAction;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.ConsentType;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.CxTypeAuthFleet;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffConsent;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffTosPrivacyActionBody;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffTosPrivacyBody;
+import it.pagopa.pn.bff.mappers.tosprivacy.TosPrivacyMapper;
+import it.pagopa.pn.bff.pnclient.userattributes.PnUserAttributesClientImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+@ContextConfiguration(classes = {TosPrivacyService.class})
+class TosPrivacyServiceTest {
+    @Autowired
+    private TosPrivacyService tosPrivacyService;
+    private PnUserAttributesClientImpl pnUserAttributesClient;
+    TosPrivacyMapper modelMapperMock = mock(TosPrivacyMapper.class);
+
+    @BeforeEach
+    void setup() {
+        this.pnUserAttributesClient = mock(PnUserAttributesClientImpl.class);
+
+        this.tosPrivacyService = new TosPrivacyService(pnUserAttributesClient);
+    }
+
+    @Test
+    void testGetTosContent() {
+        when(pnUserAttributesClient.getTosConsent(
+                Mockito.anyString(),
+                Mockito.any(CxTypeAuthFleet.class)
+        )).thenReturn(Mono.just(new Consent()));
+
+        when(pnUserAttributesClient.getPrivacyConsent(
+                Mockito.anyString(),
+                Mockito.any(CxTypeAuthFleet.class)
+        )).thenReturn(Mono.just(new Consent()));
+
+        when(modelMapperMock.mapTosPrivacyConsent(any(Consent.class)))
+                .thenReturn(new BffConsent());
+
+        StepVerifier.create(tosPrivacyService.getTosPrivacy(
+                        "UID",
+                        it.pagopa.pn.bff.generated.openapi.server.v1.dto.CxTypeAuthFleet.PF
+                ))
+                .expectNextCount(1)
```

Comment:
Use expectNext and verifyComplete as for other tests. Check all the newly added tests